README.md

```diff
@@ -80,6 +80,32 @@ let documents = vec![
 
  println!("Embeddings length: {}", embeddings.len()); // -> Embeddings length: 4
  println!("Embedding dimension: {}", embeddings[0].len()); // -> Embedding dimension: 384
+
+// reranker model usage 
```

Comment:
I recommend splitting this into a separate code block in the README.

Reply:
Ok! Should i make a new code block under this one?